### DISCUSSION TITLE
Fetching data from remote data source

### DISCUSSION THREAD

**Discussion by IrinaLuminesk** [brianvoe/slim-select#504] [General]
Thank you for creating Slim Select. My company has decided to stop using Select2 this month and we will start using your Slim Select in our future project. But in our project the data are incredibly large, my question is does your Slime Select support fetching data from a remote data source. Thank you.

**Comment by brianvoe** [comment]
Hi. In Slime Select docs under methods(https://slimselectjs.com/events#search) you can see an example of making async calls. Otherwise whatever data you put into Slime Select is what it will use to render the output.
